examples/CMakeLists.txt

```diff
@@ -112,6 +112,18 @@ function(gen_evm_verifier target input curve_type provers_amount)
         endforeach()
 
     endif()
+    file(MAKE_DIRECTORY transpiler_output_${target})
```

Comment:
Why not generate public input col in `add_example_without_proving` by assigner (new target like "prepare 
public input") and pass to transpiler instead input file?

Reply:
Yes, it would require less changes. I just don't like the idea that transpiler depends on public input, but all its actions with it is just to copy it to output directory. I would like to remove this dependency, just still have misunderstandings about the aggregate mode, maybe it can broke something.

Reply:
ok, so try to remove input file from `transpiler` cmd. If we have tests for aggregation mode in CI and change broke them - will see, otherwise we'll add `assigner` step for generation public input during enabling new tests